### PR TITLE
Dynamic Bibliography

### PR DESCRIPTION
**Pull request by susuhahnml**

### CODE REVIEW

File: krr.bib [hunk 1]
```diff
@@ -1872,6 +1876,16 @@ @article{balinski65a
   year = {1965}
 }
 
+@inproceedings{balitava20a,
+  title = {Hybrid compositional reasoning for reactive synthesis from finite-horizon specifications},
+  author = {S. Bansal and Y. Li and L. Tabajara and M. Vardi},
+  booktitle = aaai,
+  number = {06},
```
Comment (tortinator) [inline krr.bib:1883]: What's a number in a proceeding?

File: krr.bib [hunk 2]
```diff
@@ -1872,6 +1876,16 @@ @article{balinski65a
...
+  pages = {9766--9774},
+  volume = {34},
```
Comment (tortinator) [inline krr.bib:1885]: There cannot be 34 volumes in a conference proceeding  :nerd_face:

File: krr.bib [hunk 3]
```diff
@@ -1872,6 +1876,16 @@ @article{balinski65a
   year = {1965}
 }
 
+@inproceedings{balitava20a,
+  title = {Hybrid compositional reasoning for reactive synthesis from finite-horizon specifications},
+  author = {S. Bansal and Y. Li and L. Tabajara and M. Vardi},
+  booktitle = aaai,
```
Comment (tortinator) [inline krr.bib:1882]: aaai is an organization, use a crossref to refer to a conference organzied by aaai

File: krr.bib [hunk 4]
```diff
@@ -2148,7 +2162,7 @@ @inproceedings{baudinet92a
 @inproceedings{baufur94a,
   title = {Model elimination without contrapositives and its application to {PTTP}},
   author = {P. Baumgartner and U. Furbach},
-  booktitle = {Preprints Tableaux'94},
+  booktitle = {Prs Tableaux'94},
   year = {1994}
 }
```
Comment (tortinator) [inline krr.bib:2167]: A good example for a bad reference but I guess this is on me, sigh!

File: krr.bib [hunk 5]
```diff
@@ -3480,6 +3494,16 @@ @article{bogejakasc16a
...
+@inproceedings{bogigeparu20a,
+  title = {High-Level Programming via Generalized Planning and LTL Synthesis},
+  author = {B. Bonet and G. De Giacomo and H. Geffner and F. Patrizi and S. Rubin},
+  booktitle = {Proceedings of the International Conference on Principles of Knowledge Representation and Reasoning},
+  number = {1},
+  pages = {152--161},
+  volume = {17},
+  year = {2020}
+}
+
```
Comment (tortinator) [inline krr.bib:3506]: Please redo with cross reference, since kr20 should be in `procs.bib`

File: krr.bib [hunk 6]
```diff
@@ -5429,6 +5453,15 @@ @article{camaze20a
...
 
+@inproceedings{cammci19a,
+  title = {Learning interpretable models expressed in linear temporal logic},
+  author = {A. Camacho and S. McIlraith},
+  booktitle = {Proceedings of the International Conference on Automated Planning and Scheduling},
+  pages = {621--630},
+  volume = {29},
+  year = {2019}
+}
+
```
Comment (tortinator) [inline krr.bib:5464]: Please redo with cross reference, since `icaps19` should be in procs.bib

File: krr.bib [hunk 7]
```diff
@@ -10580,6 +10613,14 @@ @inproceedings{gialev99ai
...
 }
 
+@inproceedings{giarub18a,
+  title = {Automata-Theoretic Foundations of FOND Planning for {LTLf} and {LDLf} Goals.},
+  author = {G. De Giacomo and S. Rubin},
+  booktitle = ijcai,
+  pages = {4729--4735},
+  year = {2018}
+}
+
```
Comment (tortinator) [inline krr.bib:10623]: See above

File: krr.bib [hunk 8]
```diff
@@ -12237,6 +12278,13 @@ @book{hoppe95a
   year = {1995}
 }
 
+@book{hopull79a,
+  title = {Introduction to Automata Theory, Languages, and Computation},
+  author = {J. Hopcroft and J Ullman},
+  publisher = addison,
+  year = {1979}
+}
+
```
Comment (tortinator) [inline krr.bib:12287]: Thank you for moving it!

File: krr.bib [hunk 9]
```diff
@@ -19185,6 +19244,14 @@ @article{robinson65b
...
 }
 
+@book{robvor01a,
+  title = {Handbook of automated reasoning},
+  author = {A. Robinson and A. Voronkov},
+  publisher = elsevier,
+  volume = {1},
+  year = {2001}
+}
+
```
Comment (tortinator) [inline krr.bib:19254]: This is already in `procs.bib`, cf `ARHandbook`

File: krr.bib [hunk 10]
```diff
@@ -20829,7 +20905,7 @@ @inproceedings{sobamc09a
 }
 
 @article{sobanamc03a,
-  title = {Domain-Dependent Knowledge in Answer Set Planning},
+  title = {Planning with Different Forms of Domain-Dependent Knowledge in Answer Set Planning},
```
Comment (tortinator) [inline krr.bib:20908]: Nope, you are referring to another paper - be careful!

https://dl.acm.org/doi/10.1145/1183278.1183279

**Review by tortinator — CHANGES_REQUESTED** [review]

File: krr.bib (hunk 4 above)
Comment (susuhahnml) [reply krr.bib:2167]: I dont understand. I don't remeber changing this. Should there be a reference in procs.bib for `tableaux94` and reference that?

File: krr.bib (hunk 2 above)
Comment (susuhahnml) [reply krr.bib:1885]: Fixed it with the crossref

File: krr.bib (hunk 1 above)
Comment (susuhahnml) [reply krr.bib:1883]: Fixed it with the crossref

File: krr.bib (hunk 10 above)
Comment (susuhahnml) [reply krr.bib:20908]: Sorry. :(

File: krr.bib (hunk 4 above)
Comment (susuhahnml) [reply krr.bib:2164]: Perhaps in changed when I ran the format. I looked for the reference of 94 but it seams like there is not much info online. But it looks like it is the only reference anyway.

File: krr.bib [hunk 11]
```diff
@@ -10616,9 +10608,7 @@ @inproceedings{gialev99ai
 @inproceedings{giarub18a,
   title = {Automata-Theoretic Foundations of FOND Planning for {LTLf} and {LDLf} Goals.},
   author = {G. De Giacomo and S. Rubin},
-  booktitle = ijcai,
-  pages = {4729--4735},
```
Comment (tortinator) [inline krr.bib]: You have to keep the pages

**Review by tortinator — CHANGES_REQUESTED** [review]
Almost perfect :nerd_face:

**Comment by susuhahnml** [comment]
Fixed! :)

**Comment by tortinator** [comment]
Thx! FYI I deleted your branch

**Pull request merged**